### PR TITLE
feat(scripts): some fixes

### PR DESCRIPTION
**Pull request by lparet-yseop**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -10,6 +12,15 @@ Ask support to OPS team if needed.
...
 After, you can use these scripts by adding the script `docker.sh` on your repository, in the same directory as your Dockerfile.
 
+Use this command to get the script:
+
+----
+curl -O https://raw.githubusercontent.com/yseop/docker-helper/master/docker.sh
+chmod 775 docker.sh
+----
+
+Do not execute this script `docker.sh` before reading the content.
```
Comment (mminot-yseop) [inline README.adoc]: “this script” without the name (discouraged) or “the script foo.sh”, right?

File: README.adoc [hunk 2]
```diff
@@ -10,6 +12,15 @@ Ask support to OPS team if needed.
 
 After, you can use these scripts by adding the script `docker.sh` on your repository, in the same directory as your Dockerfile.
 
+Use this command to get the script:
```
Comment (mminot-yseop) [inline README.adoc]: These are _two_ commands.

File: README.adoc [hunk 3]
```diff
@@ -10,6 +12,15 @@ Ask support to OPS team if needed.
...
+
+----
+curl -O https://raw.githubusercontent.com/yseop/docker-helper/master/docker.sh
+chmod 775 docker.sh
```
Comment (mminot-yseop) [inline README.adoc]: Would `u+x` do the trick in a more secure way? I'm generally not fond of rewriting the entirety of the permission bits explicitly. It's a hassle _and_ I never really know which permissions I should give, so only adding what we need is nice.

File: docker-run.sh [hunk 4]
```diff
@@ -118,7 +117,7 @@ dock_f_start () {
 
 if [ $# -eq 0 ]
 then
-    echo "$(basename "$0"): No arguments were given; performing default operations."
+    echo "$(basename "$0"): No arguments were given;"
```
Comment (mminot-yseop) [inline docker-run.sh]: Haha, you left the semicolon but there's nothing afterwards, now. Looks weird.

File: README.adoc (hunk 3 above)
Comment (lparet-yseop) [reply README.adoc]: Agree with you. It's a bad practice that I need to change

**Comment by lparet-yseop** [comment]
@mminot-yseop Fixed, thanks for feedbacks

**Comment by mminot-yseop** [comment]
No label, no merge. :musical_note:

**Comment by lparet-yseop** [comment]
@mminot-yseop Yes but I squash :smile: I did the commit quick

**Pull request merged**